From 8513ca0e1afd109a5f9989b20409e1a417502d3a Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Wed, 22 Jun 2022 11:46:36 +0300
Subject: [PATCH] CMake: add dependency to QmlIntegration in qmlsplitlib test

Amend c3503e19e71a9fecf08a181c50d90043a4ea4edf and add dependency
to Qml to fix:

| /home/qt/work/build/build/tmp/work/core2-32-poky-linux/qtdeclarative/6.5.0-r0/git/tests/auto/qml/qmlsplitlib/lib.h:8:10: fatal error: QtQmlIntegration/qqmlintegration.h: No such file or directory
|     8 | #include <QtQmlIntegration/qqmlintegration.h>
|       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Change-Id: I591b6cc4d0d2aefc20f243a0a5b9f7780b5e47bd
---
 tests/auto/qml/qmlsplitlib/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/auto/qml/qmlsplitlib/CMakeLists.txt b/tests/auto/qml/qmlsplitlib/CMakeLists.txt
index f65b2bc7de..4a67f76da9 100644
--- a/tests/auto/qml/qmlsplitlib/CMakeLists.txt
+++ b/tests/auto/qml/qmlsplitlib/CMakeLists.txt
@@ -8,7 +8,7 @@ qt_autogen_tools_initial_setup(tst_qmlsplitlib_library)
 target_include_directories(tst_qmlsplitlib_library PRIVATE ${CMAKE_CURRENT_BINARY_DIR}) # find autogenerated header
 generate_export_header(tst_qmlsplitlib_library)
 
-target_link_libraries(tst_qmlsplitlib_library Qt::Core)
+target_link_libraries(tst_qmlsplitlib_library Qt::Core Qt::QmlIntegration)
 
 qt_internal_add_test(tst_qmlsplitlib
     SOURCES
